crates/bevy_render/src/surface/mod.rs

```diff
@@ -0,0 +1,48 @@
+mod viewport;
+
+pub use viewport::*;
+
+use crate::renderer::TextureId;
+use bevy_window::WindowId;
+
+#[derive(Debug, Copy, Clone, Hash, Eq, PartialEq)]
+pub enum SurfaceId {
+    Window(WindowId),
+    Texture(TextureId),
```

Comment:
I'm implementing an offscreeen rendering plugin on top of this, and found this a bit awkward to use.

Perhaps I should be using a Handle<Texture> here instead?